### ISSUE TITLE
On the assignment submission page, the user submission status and the result are not narrated as expected

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#349]
**Describe the bug**
On the assignment submission page, the user submission status and the result are not narrated as expected

**To Reproduce**
1. Talkback is ON
2. The student user on the Assignment overview page
3. The Submission status and the result narrated by talkback are inappropriate

**Expected behavior**
On the assignment submission page, user submission status and the result should be narrated as on the page

https://user-images.githubusercontent.com/78338755/176373915-556da27f-d870-41dc-b09d-fd05652b3e66.mp4
